migrations/core/1566290052_discrimator-multi-tenancy.up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE _core_user ADD COLUMN app_id TEXT;
+UPDATE _core_user SET app_id = '';
+ALTER TABLE _core_user ALTER COLUMN app_id SET NOT NULL;
+ALTER TABLE _core_user ADD CONSTRAINT _core_user_id_idx UNIQUE(app_id, id)
```

Comment:
I think keep using `(id)` as primary key is preferable: this can ensure IDs would not be valid across apps, in case bugs occurred in queries.

I'll change the `(app_id, id)` unique constraint to a simple index.

Reply:
Maybe the index can just be `(app_id)`?